.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Please add a `test.md` file inside `.ignored` file as you did with `node_modules` because it is  stated in your project requirement. Here the [Link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)